open_event/api/levels.py

```diff
@@ -0,0 +1,37 @@
+from flask.ext.restplus import Resource, Namespace, fields
+
+from open_event.models.session import Level as LevelModel
+from open_event.models.event import Event as EventModel
+from .helpers import get_object_list, get_object_or_404, get_object_in_event
+
+api = Namespace('levels', description='levels', path='/')
+
+level = api.model('level', {
+    'id': fields.Integer(required=True),
+    'name': fields.String,
+    'label_en': fields.String,
+})
+
+
+@api.route('/events/<int:event_id>/levels/<int:id>')
+@api.response(404, 'Level not found')
+@api.response(400, 'Object does not belong to event')
+class Level(Resource):
+    @api.doc('get_level')
+    @api.marshal_with(level)
+    def get(self, event_id, id):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2620934163/issues/source?bid=3233864&fileBranchId=3314204#l22)